### PR TITLE
Add Beacon Monitor Docs

### PR DESCRIPTION
**Pull request by Adi146**

### CODE REVIEW

File: docs/core/sensors.md [hunk 1]
```diff
@@ -272,6 +272,10 @@ There are also settings to alter:
 
 A Transmit setting toggle will start or stop the BLE transmissions - this setting is also toggled via the notification command that can turn the services on and off. 
 
+![Android](/assets/android.svg)
```
Comment (dshokouhi) [inline docs/core/sensors.md:275]: Please add a beta tag as well so users know the feature is in the beta version of the app

File: docs/notifications/commands.md [hunk 2]
```diff
@@ -142,6 +143,27 @@ automation:
             ble_transmit: "ble_transmit_high"
 ```
 
+## Beacon Monitor
+
+![Android](/assets/android.svg)
```
Comment (dshokouhi) [inline docs/notifications/commands.md:148]: Let's add a beta tag here as well since this will be in the beta first

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Just need to add the beta tags in a couple places

File: docs/notifications/commands.md [hunk 3]
```diff
@@ -22,6 +22,7 @@ The Companion apps offer a lot of different notification options. In place of po
 | `command_activity` | Launch an activity with a specified URI to any app, [more details](#activity) and use cases below. |
 | `command_bluetooth` | Turn bluetooth on or off. |
 | `command_ble_transmitter` | Turn BLE beacon transmitter on or off. |
+| `command_beacon_monitor` | Trun Beacon Monitoring on or off. |
```
Comment (dshokouhi) [inline docs/notifications/commands.md:25]:
```suggestion
| `command_beacon_monitor` | Turn Beacon Monitoring on or off. |
```

**Review by dshokouhi — APPROVED** [review]
Looks good to me, can be merged when parent is merged

**Review by JBassett — APPROVED** [review]

**Comment by JBassett** [comment]
@Adi146 Just need to fix merge conflicts

**Comment by jpelgrom** [comment]
By now the format has also changed to use `command` instead of `title` for notification commands, so not just a merge conflict to resolve.

**Comment by dshokouhi** [comment]
> By now the format has also changed to use `command` instead of `title` for notification commands, so not just a merge conflict to resolve.

Good catch totally missed that in my last review

File: docs/notifications/commands.md [hunk 4]
```diff
@@ -142,6 +143,27 @@ automation:
             ble_transmit: "ble_transmit_high"
 ```
 
+## Beacon Monitor
+
+![Android](/assets/android.svg) <span class='beta'>BETA</span><br />
+
+You can turn the Beacon Monitor on or off using `message: command_beacon_monitor` with the `title` being either `turn_off` or `turn_on`. If `title` is blank, not set or not one of the above expected values then the notification will post as normal.
```
Comment (dshokouhi) [inline docs/notifications/commands.md:150]: We need to update the format to reflect `command` in place of `title`

File: docs/notifications/commands.md [hunk 5]
```diff
@@ -142,6 +143,27 @@ automation:
...
+```yaml
+automation:
+  - alias: Turn Beacon Monitor off
+    trigger:
+      ...
+    action:
+      - service: notify.mobile_app_<your_device_id_here>
+        data:
+          message: "command_beacon_monitor"
+          title: "turn_off"
```
Comment (dshokouhi) [inline docs/notifications/commands.md:163]:
```suggestion
          data:
            command: "turn_off"
```

Lets update this format to match what is actually expected

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Just a small update for the notification command format

**Comment by Adi146** [comment]
I adjusted the docs, please let me know If I missed something

**Review by dshokouhi — APPROVED** [review]

**Pull request merged**